src/api/events.ts

```diff
@@ -7,3 +7,39 @@ export async function fetchEvents(): Promise<Array<Event>> {
   }
   throw new Error(`Failed to fetch events: ${response.statusText}.`);
 }
+
+export async function createEvent(
+  starttime: string,
+  endtime: string,
+  name: string,
+  description: string,
+  firstName: string,
+  lastName: string,
+  email: string
+): Promise<Boolean> {
+  // TODO fix url in env variable
+  const response = await fetch('https://localhost:5001/api/Event', {
+    method: 'POST', // *GET, POST, PUT, DELETE, etc.
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      starttime: starttime,
+      endtime: endtime,
+      name: name,
+      description: description,
+      owner: {
+        id: '5f747809885eeb66847e7726', //TODO remove
+        firstName: firstName,
+        lastName: lastName,
+        email: email,
+      },
+      participants: [],
+    }),
+  });
+  if (response.ok) {
+    return true;
+  }
+  return false;
+  // throw new Error(`Failed to create event: ${response.statusText}.`);
```

Comment:
This is the opposite of what I suggested for you to do during my last review 😅. The issue with this approach is that you lose the ability to tell why the request failed, e.g. is it because of network issues or is the request body incorrect/malformed?